src/Agent/Inqueue/PhoneActions.vue

```diff
@@ -45,16 +45,16 @@ export default {
       state_time: 0,
       state_date: 0,
       inqueue: null,
-      wrap_visible: false,
-      wrap: undefined
+      wrap_visible: false
     }
   },
   methods: {
     query: async function () {
       this.inqueue = await this.$agent.p_mfa('ws_agent', 'inqueue_state', ['inqueue_call', this.uuid])
     },
     queryWrap: async function () {
-      this.wrap = await this.$agent.p_mfa('ws_agent', 'inqueue_state', ['inqueue_call', this.uuid])
+      this.$agent.vm.wrap = null
```

Comment:
Why not just to pass it from the parent component?

Reply:
because dont have correct values in onwrapup state(only in oncall). When user switch between tabs and he is already in wrap up state, timer show zero value

Reply:
Why do you maintain state_time in the component?

Reply:
need to know time spent in wrapup and show correct values in wrapup timer depends on it (note there is two timers, one count down and one count up depends on time spent in wrap up)